diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/indentation/Checkers.kt

```diff
@@ -220,7 +220,7 @@ internal class DotCallChecker(config: IndentationConfig) : CustomIndentationChec
                 }
 
                 // we need to get indent before the first expression in calls chain
-                /*-
+                /* -
```

Comment:
Why?

Reply:
this is fixed diktat

Reply:
Diktat 1.2.1? Snapshot diktat already knows this style of comments `/*-` and doesn't change them